Dockerfile

```diff
@@ -23,19 +23,20 @@ ENV PIP_NO_CACHE_DIR=1
 COPY Pipfile Pipfile.lock ./
 # hadolint ignore=DL3016,DL3018,DL3013
 RUN pip install pipenv \
- && pipenv install --deploy --ignore-pipfile \
- && apk upgrade \
- && apk --no-cache add go \
-                       jq \
-                       npm \
-                       tini \
-                       ruby \
- && gem install \
-                       rubocop \
-                       rubocop-github \
- && npm install --no-cache -g dockerfile_lint \
-                       cspell \
-                       markdown-link-check
+    && pipenv install --deploy --ignore-pipfile \
```

Comment:
These formatting changes were auto-corrected on pre-commit. I'm commenting on these to increase visibility for the pre-commit addition to see if it is viable and what tweaks might be necessary in the future.